src/x25519.rs

```diff
@@ -20,22 +20,30 @@ use curve25519_dalek::constants::ED25519_BASEPOINT_TABLE;
 use curve25519_dalek::montgomery::MontgomeryPoint;
 use curve25519_dalek::scalar::Scalar;
 
-use rand_core::RngCore;
 use rand_core::CryptoRng;
+use rand_core::RngCore;
 
 /// A DH ephemeral public key.
-pub struct EphemeralPublic(pub (crate) MontgomeryPoint);
+#[derive(Copy, Clone, Debug, PartialEq)]
+pub struct EphemeralPublic(pub(crate) MontgomeryPoint);
 
 impl From<[u8; 32]> for EphemeralPublic {
     /// Given a byte array, construct an x25519 `EphemeralPublic` key
     fn from(bytes: [u8; 32]) -> EphemeralPublic {
         EphemeralPublic(MontgomeryPoint(bytes))
     }
+}
 
+impl EphemeralPublic {
+    /// Utility function to convert this public key to its raw bytes.
+    pub fn to_bytes(&self) -> [u8; 32] {
```

Comment:
Consider Tony's comment here: https://github.com/dalek-cryptography/x25519-dalek/issues/25#issuecomment-456439837

Reply:
I'm not sure if there's a good reason to have both `as_bytes` and `to_bytes`, maybe we could just have `as_bytes`?